patterns/2-structured/feature-owners.md

```diff
@@ -0,0 +1,75 @@
+## Title
+
+Feature Owners
+
+## Patlet
+
+This pattern is for scaling InnerSource project contributions aligned with business initiatives and for aspirant employees who are looking for stretch assignments, more accountability, develop their soft skills and be successful at work.
+
+## Problem
+
+Contributions in InnerSource projects mostly are all volunteer based. Below are the challenges we see in day-to-day life:
```

Comment:
```suggestion
Contributions in InnerSource projects are mostly are volunteer based. Below are the challenges we see in day-to-day life:
```

`mostly` and `all` seemed to contradict each other. I removed one, an reordered the sentence a bit. Does that work?

Reply:
Works thanks. I see two 'are's and I can remove one in the next push.